tests/git.zunit

```diff
@@ -1768,3 +2445,59 @@
     assert ${lines[5]} same_as "' file5 containing space '"
     assert ${lines[6]} same_as "\$'directory2/file6\\ncontaining\\nnewlines'"
 }
+
+@test 'Testing post: git remotes' {
+    input=(
+        'origin  git@github.com:chitoku-k/fzf-zsh-completions (fetch)'
+    )
+
+    run _fzf_complete_git-remotes_post <<< ${(F)input}
+
+
+    assert $state equals 0
+    assert ${#lines} equals 1
+
+    assert ${lines[1]} same_as 'origin'
+}
+
+@test 'Testing post: git refs' {
+    input=(
+        'origin/another-branch   2nd commit'
+        'origin/master  1st commit'
+    )
+
+    run _fzf_complete_git-refs_post <<< ${(F)input}
+
+
+    assert $state equals 0
+    assert ${#lines} equals 2
+
+    assert ${lines[1]} same_as 'another-branch'
+    assert ${lines[2]} same_as 'master'
+}
+
+@test 'Testing post: git constants ending without equal' {
+    input=(
+        '--strategy-option=subtree'
+    )
+
+    run _fzf_complete_git_git_constants_post <<< ${(F)input}
+
+    assert $state equals 0
+    assert ${#lines} equals 1
+
+    assert ${lines[1]} same_as '--strategy-option=subtree'
+}
+
+@test 'Testing post: git constants ending with equal' {
```

Comment:
This test case is inaccurate because it passes even when the output of `_fzf_complete_git_git_constants_post` ends with `\n`. Testing this is somewhat hard since a string is terminated by `\n` when it's assigned to a variable.

The following is one of the workarounds I came up with:
```zsh
@test 'Testing post: git constants ending without equal' {
    input=(
        '--strategy-option=subtree'
    )

    run _fzf_complete_git_git_constants_post <<< ${(F)input} \| tr '\n' ' '

    assert $state equals 0
    assert ${#lines} equals 1

    assert ${lines[1]} same_as '--strategy-option=subtree '
}

@test 'Testing post: git constants ending with equal' {
    input=(
        '--strategy-option=subtree='
    )

    run _fzf_complete_git_git_constants_post <<< ${(F)input} \| tr '\n' ' '

    assert $state equals 0
    assert ${#lines} equals 1

    assert ${lines[1]} same_as '--strategy-option=subtree='
}
```